src/KingSystem/Map/mapObjectLink.h

```diff
@@ -80,7 +90,25 @@ class ObjectLinkData {
 public:
     ObjectLinkData();
 
+    void deleteArrays();
     void release(Object* obj, bool a1);
+    bool allocLinksToSelf(s32 num_links, sead::Heap* heap);
+
+    bool sub_7100D4EC40(Object* src, ObjectLink* link, Object* dest);
+    void sub_7100D4FB78(Object* obj);
+    bool checkCreateLinkObjRevival() const;
+    bool checkDeleteLinkObjRevival() const;
+
+    ObjectLink* findLinkWithType(MapLinkDefType t);
+    ObjectLink* findLinkWithType_0(MapLinkDefType t);
+
+    void setGenGroup(GenGroup* group);
+
+    void x_1(act::Actor* actor, Object* obj);
+
+    bool checkCreateOrDeleteLinkObjRevival() const {
+        return checkDeleteLinkObjRevival() || checkCreateLinkObjRevival();
+    }
 
     Object* mCreateLinksSrcObj = nullptr;
     Object* mDeleteLinksSrcObj = nullptr;
```

Comment:
mNumLinksReference + mLinksReference look like they could be another sead::Buffer